src/components/elements/CommentaryBox/CommentaryBox.tsx

```diff
@@ -16,19 +24,46 @@ export interface CommentaryBoxProps {
   mutate?: any;
   refresh?: () => void;
   record?: any;
-  entity?: string;
+  entity?: AuditLogEntity;
 }
 
 const CommentaryBox = (props: CommentaryBoxProps) => {
-  const { name, lastName, buttonSendOnBox, record, entity } = props;
+  const { name, lastName, buttonSendOnBox } = props;
+  const t = useT();
+
+  const { mutate: sendCommentary } = usePostV2AuditStatusENTITYUUID({
+    onSuccess: (res: AuditStatusResponse) => {
+      const resAuditlog = res as { data: { uuid: string } };
+      const bodyFiles = new FormData();
+      files.forEach(element => {
```

Comment:
How long is this list likely to be? Might be worth looking into a bulk upload endpoint instead of hitting the server with a bunch of parallel requests if possible.